projects/plugins/boost/app/rest-api/endpoints/Request_Cloud_CSS.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Create a new request for cloud critical CSS.
```

Comment:
Can we include the HTTP method and path in the endpoint comment?

Reply:
Added